spec/go_to_param_spec.rb

```diff
@@ -99,6 +99,18 @@ def self.helper_methods
       expect(controller.go_to_here_params(bar: 3)).to eq({ go_to: "/example?foo&bar=3" })
     end
 
+    it "makes sure the request path to go to is a valid utf8 string" do
+      weird_path = "\xE0\x80\x80"
+      weird_path.force_encoding("ASCII-8BIT")
+      weird_path << "weird\330stuff".force_encoding("ASCII-8BIT")
```

Comment:
Hm, is it important that this is concatenated, to reproduce the issue? Or is it just to make the lines shorter? Not obvious to me.